FileSystem.Test/FileSystem.Test.csproj

```diff
@@ -19,6 +19,18 @@
       <None Update="Resources\file.txt">
         <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       </None>
+      <None Update="Resources\FileOpeningBuilderTest\directory\nested_file.txt">
```

Comment:
Make this a glob pattern

Reply:
Addressed in: [Use glob pattern for embedded resources](https://github.com/messerli-informatik-ag/file-system/pull/5/commits/fa5762fc3152d355e66e2f129f3d1e929aaf1543)

Reply:
Addressed in: [Allow all file types](https://github.com/messerli-informatik-ag/file-system/pull/5/commits/82f6eb76d05e32911f6cb13b3b78d0a723109ff2)